mfrc522/mfrc522.go

```diff
@@ -40,10 +41,22 @@ type config struct {
 	defaultTimeout time.Duration
 	beforeCall     func()
 	afterCall      func()
+	bogusUID       bool
 }
 
 type configF func(*config) *config
 
+// WithFixedUID sets the card reader to return correct 4-byte UIDs. Without
```

Comment:
After thinking more, I think it's better to switch the default and have the user explicitly opt in.

Reply:
I'm happy with that, it also means we wouldn't need to update either periph/cmd or periph/website to include the option in the examples.